File: JAVA/Pertemuan 01/src/id/ac/uph/pbo/pertemuan01/MobilPraktek.java
package id.ac.uph.pbo.pertemuan01;

public class MobilPraktek {

    // Atribut
    private String merek;
    private String warna;
    private int    kecepatan;

    // Constructor
    public MobilPraktek(String merek, String warna) {
        this.merek      = merek;
        this.warna      = warna;
        this.kecepatan  = 0;
    }

    // Method: jalan
    public void jalan() {
        kecepatan = 60;
        System.out.println(merek + " melaju " + kecepatan + " km/j");
    }

    // Method: berhenti
    public void berhenti() {
        kecepatan = 0;
        System.out.println(merek + " berhenti");
    }

    // Method: klakson
    public void klakson() {
        System.out.println(merek + ": Beep Beep!");
    }

    // Method: tampilkan detail
    public void info() {
        System.out.println("  Merek     : " + merek);
        System.out.println("  Warna     : " + warna);
        System.out.println("  Kecepatan : " + kecepatan + " km/j");
    }
}

File: PYTHON/Pertemuan 01/MobilPraktek.py
import sys
sys.stdout.reconfigure(encoding='utf-8')
class MobilPraktek:
    def __init__(self, merek: str, warna: str):
        self.merek = merek
        self.warna = warna
        self.kecepatan = 0

    def jalan(self):
        self.kecepatan = 60
        print(f"{self.merek} melaju {self.kecepatan} km/j")

    def berhenti(self):
        self.kecepatan = 0
        print(f"{self.merek} berhenti")

    def klakson(self):
        print(f"{self.merek}: Beep Beep!")

    def info(self):
        print(f"  Merek     : {self.merek}")
        print(f"  Warna     : {self.warna}")
        print(f"  Kecepatan : {self.kecepatan} km/j")

